config/metadata.yml

```diff
@@ -1,4 +0,0 @@
----
-version: '6.13.000'
-documentation_server: 'https://access.redhat.com'
-documentation_version: '6.13'
```

Comment:
in the past we used this field to point CI to `X.Y-beta` docs while the main version remained `X.Y`.

we didn't have to do that in the last couple of releases, so I think it's fine to drop for now (and re-introduce/re-think if  we ever gotta need this)